mainwindow.cpp

```diff
@@ -16,7 +17,7 @@ MainWindow::~MainWindow()
 
 void MainWindow::on_rockButton_clicked()
 {
-    player1Choice = ROCK;
+    player1->choice = ROCK;
     ui->paperButton->setEnabled(false);
```

Comment:
what if I want to change my choice? why we not let the button always enable?

Reply:
gud idea
imma disable the goButton until we click one of the three choice buttons
and the choice buttons are not disable

Reply:
done